Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1465,3 +1466,65 @@ void CFrame::ParseHotkeys()
 	if (IsHotkey(HK_UNDO_SAVE_STATE))
 		State::UndoSaveState();
 }
+
+void CFrame::HandleFrameSkipHotkeys()
+{
+	static const int MAX_FRAME_SKIP_DELAY = 60;
+	static int frameStep = 0;
```

Comment:
Your naming is inconsistent. `skipFrameStep` has `DelayCount` at the end of the name of its counter, but `frameStep` doesn't do that for its counter.